pandas/io/parsers/readers.py

```diff
@@ -365,17 +358,6 @@
     standard encodings
     <https://docs.python.org/3/library/codecs.html#standard-encodings>`_ .
 
-    .. versionchanged:: 1.2
-
-       When ``encoding`` is ``None``, ``errors='replace'`` is passed to
-       ``open()``. Otherwise, ``errors='strict'`` is passed to ``open()``.
-       This behavior was previously only the case for ``engine='python'``.
-
-    .. versionchanged:: 1.3.0
-
-       ``encoding_errors`` is a new argument. ``encoding`` has no longer an
-       influence on how encoding errors are handled.
```

Comment:
Accidential?

Reply:
No, 1.3 is next and this note doesn't make much sense without the note before

Reply:
I see - encoding_errors still has a versionadded below. Thanks.